package.json

```diff
@@ -18,7 +18,7 @@
     "husky": "^4.2.5",
     "npm-auth-to-token": "1.0.0",
     "typedoc-plugin-markdown": "^3.10.4",
-    "verdaccio": "4.4.4"
+    "verdaccio": "4.12.2"
```

Comment:
Asking independently from this PR: is it necessary to have these fixed dependencies here? 🤔 (so also on `npm-auth-to-token`). In doubt we will always get in new vulnerabilties in like this.

If no one knows anything about it we should rather just test to add a `^`, can't really lead to anything grave here.

Reply:
Update: I've just directly added here and will see how it goes.